governance.md

```diff
@@ -102,9 +141,11 @@ Members of the governing board will be given access to these resources:
   which is used for [test and release infrastructure](https://github.com/tektoncd/plumbing)
 * [The GCP project `tekton-nightly`](http://console.cloud.google.com/home/dashboard?project=tekton-nightly)
   which is used for publishing nightly releases for Tekton projects
-* [The GCP projects used by boskos](https://github.com/tektoncd/plumbing/blob/master/boskos/boskos-config.yaml) which are used to test against
+* [The GCP projects used by boskos](https://github.com/tektoncd/plumbing/blob/master/boskos/boskos-config.yaml)
+  which are used to test against
 
-They have the following permissions (added with https://github.com/tektoncd/plumbing/blob/master/addpermissions.py):
+They have the following permissions (added with
+https://github.com/tektoncd/plumbing/blob/master/addpermissions.py):
```

Comment:
i think we could delete the list below at this point b/c i think it's out of date and that script is probably the best source of info